benchmark/index.ts

```diff
@@ -1,12 +1,15 @@
 // eslint-disable-next-line @eslint-community/eslint-comments/disable-enable-pair -- ignore
 /* eslint-disable no-console -- ignore */
-import * as Benchmark from "benchmark";
+import Benchmark from "benchmark";
 import fs from "fs";
 import { parseForESLint } from "../src/index.js";
 import { parseForESLint as parseOld } from "../node_modules/svelte-eslint-parser/lib/index.js";
+import { fileURLToPath } from "node:url";
+import path from "node:path";
 
+const dirname = fileURLToPath(new URL(".", import.meta.url));
 const contents = `${fs.readFileSync(
-  require.resolve("../explorer-v2/src/lib/RulesSettings.svelte"),
+  path.resolve(dirname, "../explorer-v2/src/lib/RulesSettings.svelte"),
```

Comment:
Why not just `import.meta.dirname`.

Reply:
it only exists in 20.x and above

if we don't care about CI and contributors using <20, im happy to update it

Reply:
Because we need to support Node18.
https://github.com/sveltejs/svelte-eslint-parser/blob/main/package.json#L15

Reply:
Or `import.meta.resolve` for Node 20+ would be better, do we need to benchmark on old Node versions?

Reply:
The benchmark is running with `node --import tsx/esm`, I think that should be fine? Could you give it a try?

Reply:
It’s a small effort to implement support for Node 18, so I think it’s better to include it to avoid confusion.

Reply:
I mean, `import.meta.resolve` or `import.meta.dirname` should be fine when using `node --import tsx/esm`.

Reply:
it does seem to work in 18.x with `import.meta.resolve` through tsx